.flake8

```diff
@@ -1,6 +1,6 @@
 [flake8]
 # B905 should be enabled when we drop support for 3.9
-ignore = E203, E266, E501, W503, B905
+ignore = E203, E266, E501, W503, B905, B028
```

Comment:
The addition to ignore of this new option was discussed on Discord.

Reply:
Would you mind doing this in a separate PR too? We definitely need this change but I'm skeptical about the `--emoji` option.

Reply:
Done in #3502. 

Feel free to unalive this PR if you don't feel like this belongs in the codebase. I just saw the issue and it got me thinking about how hard could this be. One thing led to another and the PR appeared :smile: